api/v1/api_v1_quotas.go

```diff
@@ -123,10 +123,11 @@ func CreateIsiQuota(
 	ctx context.Context,
 	client api.Client,
 	path string, container bool, size, softLimit, advisoryLimit, softGracePrd int64,
+	includeSnapshots bool,
 ) (string, error) {
 	// PAPI call: POST https://1.2.3.4:8080/platform/1/quota/quotas
 	//             { "enforced" : true,
-	//               "include_snapshots" : false,
+	//               "include_snapshots" : true,
```

Comment:
better to retain the current behavior unless it is really required. Snapshot doesn't consume much space- maybe a few bytes for the metadata, and the size is not directly related to the source volume size.

Reply:
Writable snaphots could diverge. I do agree that we probably should not include snapshots but this is in goscaleio so the work is already done to expose this so let the applications decide on the business rules. Does not hurt to keep it.

Reply:
Although by exposing this we break compatibility. Terraform uses this module as well.